components/StationStatus.js

```diff
@@ -6,16 +6,16 @@ import {
   StatusBar,
   StyleSheet,
   Text,
-  View
+  View,
 } from 'react-native'
 import TrainList from './TrainList'
 import {
   BAR_STYLE,
   COLOR_BACKGROUND,
-  COLOR_PRIMARY
+  COLOR_PRIMARY,
 } from '../constants/styles'
 
-export default StationStatus = props =>
+export default StationStatus = props => (
```

Comment:
Possible undefined variable: 'StationStatus' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)